src/Components/Home/ListItem.js

```diff
@@ -1,59 +1,74 @@
-import React from "react";
+import { Fragment, useState } from "react";
 import { Card, Col, Row, Button, Badge } from "react-bootstrap";
 import Pages from "../PageSelection/Pages";
+import BuyDialog from "./BuyDialog";
 
 function ListItem(props) {
-  const { id, name, daysLeft, likeCount } = props.product;
+  const [showBuyDialog, setShowBuyDialog] = useState(false);
+  const { id, name, daysLeft, likeCount, questions } = props.product;
+
+  const answersCount = questions ? Object.values(questions).length : 0;
 
   const handleEditClick = () => {
     props.changePage(Pages.EDIT_PRODUCT);
     props.setCurrentProduct(props.product);
   };
 
+  const canBuy = daysLeft === 0;
+
   return (
-    <Card className="p-3" id={id} key={id}>
-      <Card.Title>{name}</Card.Title>
-      <Row>
-        <Col>
-          {daysLeft === 0 ? (
+    <Fragment>
+      <Card className="p-3" id={id} key={id}>
+        <Card.Title>{name}</Card.Title>
+        <Row>
+          <Col xs={{ span: 4 }}>
             <Button
               variant="btn btn-outline-success"
-              className="px-3"
-              onClick={() => props.removeProduct(props.product)}
+              className="px-4"
+              onClick={() => props.removeProduct(props.product, true)}
+            >
+              <i className="far fa-thumbs-up"></i> Forget
+            </Button>
+          </Col>
+          <Col xs={{ span: 8 }} className="text-right">
+            <Button
+              variant="outline-danger"
+              title="Buy product"
+              className="px-3 mb-1 mx-1"
+              onClick={() => setShowBuyDialog(true)}
+              disabled={!canBuy}
+            >
+              <i className="fas fa-shopping-cart"></i>{" "}
+              {canBuy ? "Buy" : daysLeft + " days to unlock"}
```

Comment:
for countdown message on the button, how about `daysLeft + " days to consider"` ?

Reply:
This comment can be a case for further research with users.